bin/docker-run

```diff
@@ -9,7 +9,7 @@ if [[ "${DOCKER_RUN_LOG}" == "DEBUG" ]]; then
   set -x
 fi
 
-CONTAINER_ID="${CONTAINER_ID-trinitronx/ansible-base:devel-centos7-2.3.0}"
+CONTAINER_ID="${CONTAINER_ID-trinitronx/ansible-base:devel-centos7-2.5.0}"
```

Comment:
I've been using this locally-built image for years since 2017, but just needed to update this officially as default `CONTAINER_ID` because the same image build was released & pushed via TravisCI in 2018.